packages/js-server-sdk/package.json

```diff
@@ -56,7 +56,6 @@
     "@openapitools/openapi-generator-cli": "^2.13.5",
     "@types/node": "^22.5.5",
     "@types/websocket": "^1.0.10",
-    "prettier": "3.3.3",
```

Comment:
**question:** why is prettier remove from the server sdk?

Reply:
It is now part of global package.json:
https://github.com/fishjam-cloud/js-server-sdk/pull/52/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R10